<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailSetWindow</class>
 <widget class="QDialog" name="DetailSetWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_Field0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>manufacturer:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Field1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>field1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Field2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>field2</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Field0">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>18</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Field1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>65</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Field2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>112</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Picture">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Picture:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Description">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Status">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Sale">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Sale</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_File">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Description">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>104</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Status">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>117</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Save">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
